privacy-snapshot.json

```diff
@@ -6,19 +6,21 @@
   "api.segment.io",
   "api.simplehash.com",
   "api.testnet.solana.com",
+  "api.web3auth.io",
   "api.web3modal.com",
   "api.web3modal.org",
   "app.ens.domains",
   "arbitrum-mainnet.infura.io",
+  "auth-service.dev-api.cx.metamask.io",
```

Comment:
Any idea how this got here? Does social login rely on the auth service in some way?

Reply:
Yes. Auth service is a new server built by our team which manages the social logins